### ISSUE TITLE
Feature Request : JavaScript and CSS Code Beautifier Modules

### ISSUE THREAD

**Issue by Gobinath-B** [pratyushtiwary/devty#38]
###Description

. Add JavaScript and CSS Code Beautifier Modules that helps developers to make the messy js and css code from browser and other sources understandable.
. We can add json viewer functionality too

**Comment by pratyushtiwary** [comment]
@Gobinath-B are you willing to add these modules?

You can add [prettier](https://prettier.io) if you want to.

**Comment by Gobinath-B** [comment]
yeah sure, I want to add prettier module. Can you assign me on this work?

**Comment by pratyushtiwary** [comment]
> yeah sure, I want to add prettier module. Can you assign me on this work?

sure
